lib/request-fragment.js

```diff
@@ -4,16 +4,21 @@ const http = require('http');
 const https = require('https');
 const url = require('url');
 const filterHeaders = require('./filter-headers');
+const defaultHeaders = {
+    'accept-encoding': 'gzip, deflate'
+};
 
 module.exports = function requestFragment (fragmentUrl, fragmentAttributes, request) {
     return new Promise((resolve, reject) => {
         const parsedUrl = url.parse(fragmentUrl);
         const options = Object.assign({
-            headers: filterHeaders(fragmentAttributes, request.headers),
+            headers: Object.assign(
+                filterHeaders(fragmentAttributes, request.headers),
+                defaultHeaders
```

Comment:
This overwrites possibly filtered headers with the default headers. I understand, that this is what you want, but it breaks my expectation of "defaultHeaders". I think it should be renamed to something like "requiredHeaders" since they are always there no matter what filterHeaders sets.